scripts/breaking_changes_checker/breaking_changes_allowlist.py

```diff
@@ -8,7 +8,6 @@
 
 RUN_BREAKING_CHANGES_PACKAGES = [
     "azure-ai-formrecognizer",
-    "azure-ai-textanalytics"
```

Comment:
Is this relevant? No problem being in here, but wondering if this was just something you noticed or what. Mainly confused because you're adding APIs but removing a breaking change...thing.

Reply:
I tried to comment on that line but it didn't show up for some reason. This is a legacy breaking changes checker (which I wrote 😆) that we don't really use anymore. It was raising a false positive on `is_error` saying that I removed it from the API so I'm disabling it.